3D-microservices-observability/node-js-app/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "beachshirt",
+  "version": "1.0.0",
+  "description": "beachshirt application which makes cool shirts for the beach.",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
Is this the OSS license that VMWare legal likes?

Reply:
removed the license from here as we are already having it in the parent dir